dashboard/app/models/user.rb

```diff
@@ -168,6 +168,7 @@ class User < ApplicationRecord
     has_seen_ai_assessments_announcement
     seen_ta_scores_map
     roster_synced
+    educator_type
```

Comment:
In [the product spec](https://docs.google.com/document/d/1a_RxfC2CTVLIoc__8X375EjhSjCHyaLxPRZAzsg4m1Y/edit?tab=t.ggy8fsnwj17a), this is called `educator_role`, so I may slightly prefer that name. Don't have a strong opinion, though, if others have thoughts!

Reply:
`educator_role` sounds good to me, I was just going off of what I jotted down from our meeting, which I figured very well might be wrong. I'll update it.